exo-lint/src/cli.ls

```diff
@@ -11,8 +11,5 @@ module.exports = ->
   app-config = yaml.safe-load fs.read-file-sync('application.yml', 'utf8')
   console.log "Running linter for #{cyan app-config.name}\n"
   logger = new Logger Object.keys(app-config.services)
-  app-linter = new AppLinter app-config
-    ..on 'lint-success', -> logger.log name: 'exo-lint', text: 'Lint passed'
-    ..on 'output', (data) -> logger.log data
-    ..on 'reset-colors', (service-names) -> logger.set-colors service-names
```

Comment:
The line that was removed is actually unneeded. When initializing the logger with `Object.keys (app-config.services)`, colors are assigned for each of the services. The event that was emitted to reset the colors was passing the same `Object.keys (app-config.services)`, so there was really no reason for it to be there.